common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockStoreClient.java

```diff
@@ -46,6 +46,8 @@
 
   protected volatile TransportClientFactory clientFactory;
   protected String appId;
+  // Store the application attemptId
+  protected int appAttemptId;
```

Comment:
`protected` -> `private` ?

Reply:
Or rather, move it to ExternalBlockStoreClient and keep it private there ?

Reply:
I actually keep the appAttemptId within BlockStoreClient and move the setAppAttemptId into the BlockStoreClient, as in BlockManager it only calls from BlockStoreClient, other than ExternalBlockStoreClient.

Reply:
If we are moving it here, make it `private`

Reply:
Moved to private.